packages/tuono/src/config/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface TuonoConfig {
```

Comment:
To mimic `vite` structure I would consider to name this `TuonoUserConfig`.
Maybe in the future we need to perform some normalization and we mind end up to having a `TuonoNormalizedConfig` or something similar.

Reply:
I don't like it, honestly. I prefer just `TuonoConfig` like `NextConfig`